kernel/kata_config_version

```diff
@@ -1 +1 @@
-31
+33
```

Comment:
I am sorry the file already has `33` better you rebase master and then bump to `34`.